plugins/aws-s3-storage/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM verdaccio/verdaccio:4
+
+USER root
+
+ENV NODE_ENV=production
+
+RUN apk --no-cache add openssl ca-certificates wget && \
```

Comment:
This is a personal thought, I used this in the past for some reason but I don't know whether is need, in case is need it, why? People ask me but I have no answer, do you know why?

Reply:
honestly I took this straight away from the examples repo.

Regarding python and some g++ stuff, I can for sure say that aws sdk needs them, because they use some python binding.

Reply:
Thanks :)